conda_env.yml

```diff
@@ -1,4 +1,4 @@
-name: stream
+name: stream2
```

Comment:
why change the environment name?

Reply:
Environment name change was accidental - I changed it locally to test a new environment without clobbering my existing one and forgot to change it back. The 3rd commit restores the original.